tls/s2n_config.c

```diff
@@ -229,7 +229,7 @@ int s2n_config_add_cert_chain_and_key(struct s2n_config *config, const char *cer
     GUARD(s2n_alloc(&mem, sizeof(struct s2n_cert_chain_and_key)));
     config->cert_and_key_pairs = (struct s2n_cert_chain_and_key *)(void *)mem.data;
     config->cert_and_key_pairs->head = NULL;
-    config->cert_and_key_pairs->private_key.rsa = NULL;
+    config->cert_and_key_pairs->private_key.ctx = NULL;
```

Comment:
Why not also preserve the removed line with:
`config->cert_and_key_pairs->private_key.rsa_key.rsa = NULL;`